README.md

```diff
@@ -181,6 +194,25 @@ exporters:
     endpoint: "http://127.0.0.1:9411/api/v2/spans"
 ```
 
+### <a name="config-pipelines"></a>Pipelines
+Pipelines can be of two types:
+
+- metrics: collects and processes metrics data.
+- traces: collects and processes trace data.
+
+A pipeline consists of a set of receivers, processors, and exporters. Each receiver/processor/exporter must be specified
```

Comment:
Formatting suggestion: either stick to soft line breaks (i.e. only do hard line breaks between paragraphs) or use hard line breaks at around 80 characters in MD files. With longer lines (like 120 chars in this case) the side-by-side diff in Github doesn't look as nice unless you have a very large monitor.

Reply:
Sounds good - I'll update.

To make things easier to catch especially with formatting, can we add a md formatter to catch all of these things?

Reply:
There is an ongoing discussion on this topic: https://github.com/open-telemetry/opentelemetry-specification/issues/192

After the decision is made we can look into having an md formatter to enforce the rules.

Reply:
Great! That will be helpful to save everyone time and even have it as a precommit check!